plugins/core/webview/src/q-ui/components/loginOptions.vue

```diff
@@ -2,8 +2,8 @@
 <!-- SPDX-License-Identifier: Apache-2.0 -->
 
 <template>
-    <QOptions v-if="app === 'AMAZONQ'" @stageChanged="stageChanged" @login="login"/>
-    <ToolkitOptions v-if="app === 'TOOLKIT'" @stageChanged="stageChanged" @login="login"/>
+    <QOptions v-if="app === 'AMAZONQ'" @stageChanged="stageChanged" @login="login" @sendTelemetry="sendTelemetry"/>
```

Comment:
we could just reuse @login maybe?

Reply:
https://github.com/aws/aws-toolkit-jetbrains/blob/main/plugins/core/webview/src/q-ui/components/login.vue#L83-L104

Reply:
so that we don't need to add @sendTelemetry event in qLoginOptions.vue & toolkitLoginOptions.vue

Reply:
btw i think you missed toolkitLoginOptions.vue